### PR TITLE
docs: add instructions for changing theme

### PR DESCRIPTION
**Pull request by lakshyaag**
- **Description:** Add instructions on how to change color, font family, and styles
- **Issue:** #12 (partially)
- **Dependencies:** N/A

### CODE REVIEW

**Review by tianjing-li — APPROVED** [review]
lgtm @elaineg can you check this to make sure its in line with what you're looking for?

File: README.md [hunk 1]
```diff
@@ -432,6 +432,38 @@ Toolkit includes some sample tools that you can copy to configure your own data
 
 To create your own tools or add custom data sources, see our guide: [tools and retrieval sources overview](/docs/custom_tool_guides/tool_guide.md)
 
+
+## Themeing and customization
```
Comment (wujessica) [inline README.md:436]:
```suggestion
## Theming and customization
```

File: README.md [hunk 2]
```diff
@@ -432,6 +432,38 @@ Toolkit includes some sample tools that you can copy to configure your own data
...
+- Modify the `src/interfaces/coral_web/src/themes/cohereTheme.js` with a new color scheme. For example, to change the primary color scheme:
+
+```js
+primary: {
+          ...
+          600: '#E25D41',
+          500: '#AE4359', // Changed from default color
+          400: '#FF967E',
+          ...
+        },
```
Comment (wujessica) [inline README.md:451]:
```suggestion
  ...
  600: '#E25D41',
  500: '#AE4359', // Changed from default color
  400: '#FF967E',
  ...
},
```

**Review by wujessica — APPROVED** [review]
Look great! Just left some tiny nits, thanks for making these changes!!

**Comment by lakshyaag** [comment]
Thanks @wujessica, I've made the suggested changes. Not sure what to do with the merge conflict here.

**Comment by wujessica** [comment]
> Thanks @wujessica, I've made the suggested changes. Not sure what to do with the merge conflict here.

Ahh I guess we've restructured our README.md, I think this could go under [`how_to_guides.md`](https://github.com/cohere-ai/cohere-toolkit/blob/main/docs/how_to_guides.md)? Would you mind moving it into that file and following that format? And adding a link on L13 in the README.md file

**Review by wujessica — APPROVED** [review]
nice, thank you so much!!

**Comment by lakshyaag** [comment]
Great, please merge when ready. 
@wujessica, any comments on the logo part of #12? I'll try creating a prototype tonight

**Comment by wujessica** [comment]
> Great, please merge when ready. @wujessica, any comments on the logo part of #12? I'll try creating a prototype tonight

That approach makes sense to me, excited to see the prototype :)

**Comment by wujessica** [comment]
@lakshyaag looks like some of the commits on this PR aren't signed so I can't merge, could you help sign them?

image

**Comment by lakshyaag** [comment]
Hi @wujessica, I just added the GPG key locally. Could you please help me on how I can go back and sign all the commits?

**Comment by wujessica** [comment]
@lakshyaag you might need to squash and re-commit, it looks like it's your earlier commits that weren't signed (any of the below that don't say `Verified`)

image

**Comment by lakshyaag** [comment]
I think I messed up. Let me re-open this with a clean version.

**Pull request closed without merging**